src/test-run/index.js

```diff
@@ -148,6 +162,25 @@ export default class TestRun extends Session {
         return new Promise((resolve, reject) => this.pendingDriverTask = { command, resolve, reject });
     }
 
+    _addPendingWindowTask () {
+        var command = new ExecutePendingBrowserManipulationCommand();
```

Comment:
ok if that's what it actually does.
So, `PrepareBrowserManipulationCommand` for the command and `prepare-browser-manipulation` for the executor